requirements/requirements.txt

```
@@ -1,19 +1,19 @@
 colorama
```

Without pinning `brax`, as well MaBrax can also fail. 
```suggestion
brax==0.10.3
colorama
```